modules/fedimint-lnv2-client/src/lib.rs

```diff
@@ -515,8 +515,20 @@ impl LightningClientModule {
             .await
     }
 
-    /// Pay an invoice. For  testing  you can optionally specify a gateway to
-    /// route with, otherwise a gateway will be selected automatically.
+    /// Pay an invoice. For  testing you can optionally specify a gateway to
+    /// route with, otherwise a gateway will be selected automatically. If the
+    /// invoice was created by a gateway connected to our federation this
+    /// gateway will be selected, otherwise we select a random online gateway.
+    ///
+    /// The fee for this payment may depend on the selected gateway but
+    /// will be limited to one and a half percent plus one hundred satoshis.
+    /// This fee accounts for the fee charged by the gateway as well as
+    /// the additional fee required to reliably route this payment over
+    /// lightning if necessary. Since the gateway has been vetted by at least
+    /// one guardian we trust it to set a reasonable fee.
```

Comment:
Does this mean that you expect gateways to not change fees after being vetted?

Reply:
No, but it does mean that guardians check up on the gateways fees (by using the federation themselves for example) according to how much they trust them. If they start out with our gateway for example, or have a gateway that had good service and fees for months, the check ups are less frequent.

Reply:
I just don't think in practice most guardians are going to be consistently verifying fees and other routing parameters. Clients can always route around vetted gateways by just providing the URL endpoint to `send`, and this may not have been vetted by the guardians, so the comment doesn't seem 100% applicable to me.